recipe/thirdparty/providers/custom.go

```diff
@@ -27,21 +27,75 @@ func NewProvider(input tpmodels.ProviderInput) tpmodels.TypeProvider {
 		ID: input.ThirdPartyID,
 	}
 
```

Comment:
can add userInfoMap for id token:
- userId is sub
- email is email
- email_verified is email_verified

Reply:
Done